compiler/fir/cones/src/org/jetbrains/kotlin/fir/utils/ArrayMapOwner.kt

```diff
@@ -70,6 +70,6 @@ abstract class TypeRegistry<K : Any, V : Any> {
         return idPerType.computeIfAbsent(kClass) { idCounter.getAndIncrement() }
     }
 
-    protected val indices: Collection<Int>
+    internal val indices: Collection<Int>
```

Comment:
This is also not needed due to previous comment

Reply:
Right, reverted.